roles/devices/templates/zebrunner-farm

```diff
@@ -66,6 +66,26 @@ function create_containers () {
     local wireless=`cat ${devices} | grep "$udid" | cut -d '|' -f 9`
     #echo "wireless: $wireless"
 
+    local server_proxy_port=`cat ${devices} | grep "$udid" | cut -d '|' -f 12`
+    #echo "server_proxy_port: $server_proxy_port"
+
+    local certificates_proxy_path={{ CA_CERTS_PROXY_PATH }}
```

Comment:
let's try to remove CA_CERTS_PROXY_PATH  usage and switch to the docker volume which is shared across all mitm containers.
item to test: how ca certificates generated/regenerated during 2nd, 3rd etc device start

Reply:
done